### DISCUSSION TITLE
ddns go与dynv6四级域名

### DISCUSSION THREAD

**Discussion by yrhy170** [jeessy2/ddns-go#1306] [Q&A]
在dynv6中创建了一个四级域名
![image](https://github.com/user-attachments/assets/79f1b0e0-809e-44e5-b35a-deda937ff10e)
并获得了dynv6的token
![image](https://github.com/user-attachments/assets/1d34bc3b-b067-42b0-989e-fb59bf6f5c04)
在ddns填入token并这样设置
![image](https://github.com/user-attachments/assets/1d375deb-a85b-435a-b18b-b093c7e52a97)
Callback调用失败, 异常信息: 返回内容: zone not found ,返回状态码: 404
![image](https://github.com/user-attachments/assets/e9e23dd8-a819-4cba-befd-c8d374513b1c)
该怎么办呢
这大概不应该在这里问
但我能问的都问过了:(

**Comment by yrhy170** [comment]
这样
![image](https://github.com/user-attachments/assets/cb5fb7bd-977d-4988-9c84-beb8e131de5d)
也试过了

**Comment by yrhy170** [comment]
请问是不支持四级域名吗

**Comment by jeessy2** [comment]
你要问dynv6

**Reply by yrhy170** [reply]
o
